app/views/smart_proxies/show.html.erb

```diff
@@ -69,5 +73,18 @@
         </div>
       </div>
     <% end %>
+    <% tab_features.each do |feature| %>
+      <div class="tab-pane" id="<%= feature %>">
+      <% feature_erb = "smart_proxies/plugins/#{feature}" %>
+      <% if lookup_context.template_exists?(feature_erb, [], true) %>
+        <%= render :partial => feature_erb %>
+      <% else %>
+        <div class="col-md-12">
+          <%= render :partial => "smart_proxies/plugins/no_template", :locals => {:feature => feature} %>
+        </div>
+      <% end %>
+      </div>
+    <% end %>
+    </div>
```

Comment:
Now I understand why you tried to squeeze everything into one tab. Well, I think we need to get rid of iterating though features in order to get a usable UI. Or find a better way.